kotlinx-coroutines-core/common/src/CoroutineScope.kt

```diff
@@ -292,24 +315,41 @@ public fun CoroutineScope.cancel(cause: CancellationException? = null) {
 public fun CoroutineScope.cancel(message: String, cause: Throwable? = null): Unit = cancel(CancellationException(message, cause))
 
 /**
- * Ensures that current scope is [active][CoroutineScope.isActive].
+ * Ensures that current scope is [active][CoroutineScope.isActive] and throws the [CancellationException] that was the
+ * scope's cancellation cause otherwise.
```

Comment:
```suggestion
 * Throws the [CancellationException] that was the scope's cancellation cause if the scope is no longer [active][CoroutineScope.isActive].
```
Otherwise, it may leave the impression that "ensuring the scope is active" is some procedure that's not just throwing an exception.